tests/integration/records/table_validator.py

```diff
@@ -112,7 +112,19 @@ def format_type(column: Dict[str, Any]) -> str:
             return str(column['type']) + suffix
 
         actual_column_types = [format_type(column) for column in columns]
-        assert actual_column_types in expected_column_types, actual_column_types
```

Comment:
This file, one of the 3 largest in the project, increased in size to 423 lines.  The total size of those files is now 3465 lines (target: 897).  Is this file complex enough to refactor?